docs/cuj/answers/exercise4/checkout-pipeline.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: pipeline.knative.dev/v1alpha1
+kind: Pipeline
+metadata:
+  name: microservices-pipeline
+  namespace: default
+spec:
+    tasks:
+      - name: test-checkout
+        taskRef:
+          name: go-test
+        inputSourceBindings:
+          - name: app-github
```

Comment:
could you remove the `name` fields from the bindings? hopefully will be going away in #191 anyway :)